mmeds/resources/log_config.yaml

```diff
@@ -0,0 +1,26 @@
+version: 1
+formatters:
+    default:
+        format: '%(levelname)s - %(message)s'
+handlers:
+    console:
+        class: logging.StreamHandler
+        formatter: default
+        level: DEBUG
+    file:
+        class: logging.handlers.RotatingFileHandler
+        level: DEBUG
+        formatter: default
+        filename: /home/david/Work/ensemble/logging/ensemble_debug.log
```

Comment:
I'm pretty sure this is being set somewhere, and could therefore be blank. Might be better that way to key people off that `filename` is being set programmatically.

Reply:
tis